choreographer/browser.py

```diff
@@ -345,6 +347,16 @@ async def _async_close(self):
 
     def close(self):
         if self.loop:
+            for future in self.futures:
+                future.set_exception(BrowserClosedError())
+            for session in self.sessions:
```

Comment:
here can be better with Porotocol.sessions? I just say because protocol have all the sessions, or is right with Browser.sessions like in your code?

Reply:
do you think they would ever be different? should we check both?